src/query/plan/operator.cpp

```diff
@@ -201,6 +201,15 @@ std::vector<storage::LabelId> EvaluateLabels(const std::vector<StorageLabelType>
   return result;
 }
 
+storage::EdgeTypeId EvaluateEdgeType(const StorageEdgeType &edge_type, ExpressionEvaluator &evaluator,
```

Comment:
Aha, one implication of this whole PR is users can more easily create a bunch of EdgeTypeIDs (I mean, it's not that hard even with the "static" EdgeType). Should we update documentation or put some kind of a limit on how many edge types is expected to be present in a given graph? 🤔

Reply:
Move this also to the PR description.